openedx/core/djangoapps/user_authn/views/login.py

```diff
@@ -56,27 +56,17 @@ def _do_third_party_auth(request):
             u"with backend_name {backend_name}".format(
                 username=username, backend_name=backend_name)
         )
-        message = _(
+        message = Text(_(
             u"You've successfully logged into your {provider_name} account, "
```

Comment:
We try to use "sign in" instead of "log in", so this could be "You've successfully signed in to...

Reply:
Done, thank you @grantgoodmanedX!
https://github.com/edx/edx-platform/pull/20253